lib/ansible/utils/module_docs_fragments/docker.py

```diff
@@ -67,9 +67,18 @@ class ModuleDocFragment(object):
         description:
             -  Secure the connection to the API by using TLS without verifying the authenticity of the Docker host
                server.
+        default: false
+        type: bool
     tls_verify:
         description:
             - Secure the connection to the API by using TLS and verifying the authenticity of the Docker host server.
+        default: false
+        type: bool
+    debug:
```

Comment:
Make sure to remove this from the module docs now that it is present in the docs fragment.